src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,290 @@
+
+Now that the issue is in the right component and has the basic information, the analysis continues to get a more detailed understanding of the issue, and what should be done:
+
+1. ensure the priority is correct - an approach that has been used for getting a consistent view of priority is to consider three aspects of the issue: [Impact]{.jbs-field} or issue; [Likelihood]{.jbs-field} of it occurring; and, whether there is a [Workaround]{.jbs-field}. The higher the [Impact]{.jbs-field} and [Likelihood]{.jbs-field} the higher the priority; then, having a [Workaround]{.jbs-field} reduces that priority - but mostly where the Impact and Likelihood are not that severe.
+1. ensure the 'Affects Version/s' field is correct (within reason)
+    1. this may involve reproducing the bug, if doing so is fast and easy
+    1. In addition to the version where the bug was found, take special care to also investigate if the bug affects the supported releases - the latest LTS release and the latest six-month release.
+    1. The affects version is not meant to be an exhaustive list of releases the issue is relevant to - it should initially be set to the release the issue was reproduced or identified on, and by implication it will be relevant on all releases past that point (see (Rel)-na label). If it is later found to be applicable to an earlier release family then adding that earlier release is encouraged if the issue needs to be fixed in that release.
+    1.  Do not add additional release values to `Affected Versions` for the same release family: if there is the value 11.0.2 do not add 11.0.5, 11.0.7 etc.  Adding an additional value for a separate release family where it is still reproducible, 12 say, is not necessary but ok, especially if the bug is old (reported on 8 say) but still relevant to the latest release (20 say).
+1. set the 'Fix Version'
+    1. a bug should be fixed first in the most recent version where it exists, if you don't know what version it will go into set it to `tbd`
+    1. if the bug also exists in older versions it may require backporting
+        * the decision to backport should be made inline with the guidelines of the lead for the release
+        * There are two options for creating backport issue(s) to track the backport: one is to create it manually once it is agreed that the bug should be backported; the second, is to let the bots create the backport issue once you push the fix to the repo.
+    1. Only one fixversion should ever be set, if the issue is to be fixed in additional releases then a separate backport must be created (See Working with backports in JBS).  There are exceptions to this rule for: CSRs; and, Release Notes.
+1. make sure the bug has all the required labels – JBS Label Dictionary
+    1. bugs that are new in the current release: 'regression'
```

Comment:
We probably should be better at marking that last category as regressions though since there are special rules around deferring those.

Reply:
Have left the text as-is for the moment - let me know if you feel changes are needed

Reply:
> This is tricky. We don't generally use "regression" during active development of the current release, it tends to be used mainly for issues that are found after a release has been released e.g. if we release JDK 19 and then a bug report comes in because something that worked in 18 now doesn't, then it is a regression in 19. But if we are in the process of developing a release and we have added some new thing and then we find a bug in that then we don't mark that as a regression. And a lot of the time if we introduce a bug "by mistake" we don't tend to mark it as a regression.

Right; if something didn't "work" in a shipped release, it can be a bug, but *not* a regression.

Reply:
updated the definition of a regression to be:

A bug is a regression when behavior has _incorrectly_ changed from a previous release.